src/tests/testUtils.tsx

```diff
@@ -0,0 +1,27 @@
+import { render } from "@testing-library/react"
+import { ThemeProvider } from "@emotion/react"
+import { theme } from "../theme"
+import { Provider } from "react-redux"
+import store from "../redux"
+import React from "react"
+import { Dashboard } from "../apps/Dashboard"
+import { createHashHistory } from "history"
+
+window.location.hash = "/"
+const renderDashboardApp = (path: string) => {
+  const history = createHashHistory()
+  history.push(path)
+
+  return {
+    ...render(
+      <ThemeProvider theme={theme}>
+        <Provider store={store}>
+          <Dashboard />
+        </Provider>
+      </ThemeProvider>
+    ),
+    history,
+  }
+}
+
+export default renderDashboardApp
```

Comment:
I think this file is obsolete with the integration tests, no?

Reply:
yeah. good point. since all integration test cases are in cypress we can delete this.
Previously, I had assumed to do integration tests through jest but not required anymore.
Will get it remove over next weekend. 👍

Reply:
updated 👍